staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -1114,6 +1118,12 @@ func (r Result) StatusCode(statusCode *int) Result {
 	return r
 }
 
+// Header returns the HTTP header of the response.
```

Comment:
nit: Function doesn't return the header, although I do see the above godoc for StatusCode is also phrased this way. Change both to a form like "Header sets the provided header to the HTTP header of the response"?

Reply:
thanks. Updated